router/router.go

```diff
@@ -114,4 +114,6 @@ func Docker(group *echo.Group, reg registry.Registry) {
 
```

Comment:
Also worth noting that the Function here called Docker can totally be called `Extensions`